### DISCUSSION TITLE
CORS configuration in Microcks

### DISCUSSION THREAD

**Discussion by ghost** [microcks/microcks#976] [Ideas]
How do we configure CORS in microcks? (globally, per mock, per request) Or all of them?

**Comment by lbroudoux** [comment]
Origin of discussion comes from here: https://github.com/microcks/microcks/issues/956#issuecomment-1790959695

**Comment by ghost** [comment]
For a Microcks Service / API version - think of the inclusion of an extra attribute in x-microcks specific attribute or other.
    -- in that case, it won't probably fit all frontend clients needs 
By default we could use a dynamic cors filter where the origin is the same as the request. 
Other cors parameters should be set as Default from the global microcks cors configuration.

In an spec file in extensions we could add in info/x-microcks:
```yaml
cors:
    - origins: "<origins>"
    - allowCredentials: "true|false"
    ...
```
